aws/sdk/examples/autoscaling/src/bin/create-autoscaling-group.rs

```diff
@@ -46,21 +46,24 @@ async fn main() -> Result<(), Error> {
         verbose,
     } = Opt::from_args();
 
-    let region_provider = region::ChainProvider::first_try(region.map(Region::new))
+    let region = region::ChainProvider::first_try(region.map(Region::new))
```

Comment:
We could call this `region`, but it really is a region `ChainProvider` instance. Are we worried at all that people might mistake it for a `Region` instance when calling it `region`?